contracts/utils/ExchangeRate.sol

```diff
@@ -1,26 +1,36 @@
-//SPDX-License-Identifier: Unlicense                                                                                                                                         [0/0]
+//SPDX-License-Identifier: Unlicense
 pragma solidity ^0.6.12;
 
 import "@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol";
-/* exchangeRate = principal + surplusInterest / bTokenTotalSupply */
 
+/* @title exchange rate calculation */
+/* @notice from bTokens to aTokens and vice versa */
+/* @dev the equation is exchangeRate = principal + surplusInterest / bTokenTotalSupply */
 contract ExchangeRate {
 
-    // calculates token from btoken amount
+    // @notice returns tokens from btokens, pure function
+    /* @param _principal amount in underlying asset deposited into the contract  */
+    /* @param _surplus the interests earned that belong to the depositorReserve */
+    /* @param _btokens the depositor's btokens amount to convert to underlying tokens */
+    /* @return amount in underlying tokens to return to depositor */
     function btokenToToken(uint256 _principal,
                            uint256 _surplus,
                            uint256 _tokenSupply,
-                           uint256 btokens
+                           uint256 _btokens
                            ) pure public returns (uint256){
-        return SafeMath.div(SafeMath.mul(btokens, SafeMath.add(_surplus, _principal)), _tokenSupply);
+        return SafeMath.div(SafeMath.mul(_btokens, SafeMath.add(_surplus, _principal)), _tokenSupply);
     }
 
-    // calculates btokens amount from tokens
+    // @notice returns btokens from tokens, pure function
+    /* @param _principal amount in underlying asset deposited into the contract  */
+    /* @param _surplus the interests earned that belong to the depositorReserve */
```

Comment:
want to add an `@param` doc for _tokenSupply?

Reply:
yes, good call